include/CCRing.h

```diff
@@ -44,6 +39,11 @@ CCError freeRing(CCRing* pRing);
 records a pointer where the last change was made to the Ring
 and appends at the position of the pointer with each new
 append. */
-int ccAppend(CCRing* appendRing, ccAudioDataType array[], unsigned long length);
```

Comment:
I can't remember - did we decide to rename `ccApend` to `ringAppend`? Looking at it now, `ccAppend` looks nicer. That also follows our conventions of prefixing function names that operate on a ring with `cc`.